PIL/Image.py

```diff
@@ -1146,6 +1212,7 @@ def getbbox(self):
         return self.im.getbbox()
 
     def getcolors(self, maxcolors=256):
+        # type: (int) -> Optional[List[Tuple[int, int]]]
```

Comment:
Ah, so it's that the parameter is always an int, not that it's sometimes required as a named parameter.